### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by mknaranja**

### CODE REVIEW

**Comment by mknaranja** [comment]
@xsaschako Please have a look and merge if possible :)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```
Comment (xsaschako) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]: maybe add an e.g.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: class::bug
+assignees: ''
+
+---
+
+**Describe the bug**
```
Comment (xsaschako) [inline .github/ISSUE_TEMPLATE/bug_report.md:10]: All subtitles should have a line break after them.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```
Comment (xsaschako) [inline .github/ISSUE_TEMPLATE/feature_request.md:10]: No punctuation for consistency

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (xsaschako) [inline .github/ISSUE_TEMPLATE/feature_request.md:11]: Ex -> E.g.

**Review by xsaschako — COMMENTED** [review]
Looks good. Some minor nitpicks.
Also i don't like the table at the top, that looks a bit odd. Maybe you can even delete it.

**Review by xsaschako — APPROVED** [review]
Looks good!

**Pull request merged**